packages/kcmsf/src/routes/router.tsx

```diff
@@ -26,6 +27,10 @@ export const router = createBrowserRouter(
           path: "entry",
           element: <Entry />,
         },
+        {
+          path: "entrybulk",
+          element: <EntryBulk />,
+        },
```

Comment:
```suggestion
        {
          path: "entry",
          children: [
            {
              index: true,
              element: <Entry />,
            },
            {
              path: "bulk",
              element: <EntryBulk />,
            },
          ],
        },
```
書き忘れていました。こんな感じに書くと`/entry/bulk`を使えるようになります。